int16 CH1
int16 CH2
int16 CH3
int16 CH4
int8 emergency
int8 AUX